lib/splitclient-rb/clients/split_client.rb

```diff
@@ -148,6 +154,9 @@ def store_impression(split_name, matching_key, bucketing_key, treatment, store_i
         'time' => time,
         'changeNumber' => treatment[:change_number]
       )
+    rescue StandardError => error
+      @config.log_found_exception(__method__.to_s, error)
+      @config.logger.warn('Failed to store impression')
```

Comment:
did you forget to remove this?

you need to pass `Failed to store impression` to the method above or another way to provide a context message